battleship.py

```diff
@@ -59,6 +64,10 @@ def keyPressed(data, event):
 Returns: None
 '''
 def mousePressed(data, event, board):
+    click =getClickedCell(data,event)
+    if board == "user" :
+     clickUserBoard(data, click[0], click[1])
+    
     pass

```

Comment:
remove pass

Reply:
Also, resolve conflicts by deleting pycache and then clicking on the resolve conflicts button for battleship.py

Reply:
issue resolved